parser/parse.go

```diff
@@ -85,27 +89,42 @@ func ParseDns(handle *pcap.Handle) {
 	packetSource.NoCopy = true
 
 PACKETLOOP:
-	for packet := range packetSource.Packets() {
+	for {
 		stats.PacketTotal += 1
 
+		parser := gopacket.NewDecodingLayerParser(layers.LayerTypeEthernet, &ethernet, &ip4, &ip6, &tcp, &udp, &dot1q)
+		parser.IgnoreUnsupported = true
```

Comment:
Please don't drop the `UnsupportedLayer()` errors. These are useful for debugging things later.